### ISSUE TITLE
Possibility to import those corrupted ? files

### ISSUE THREAD

**Issue by lpatiny** [image-js/tiff-js#5]
An instrument generates the following uncompressed files that seems however problematic
[Eimg_000000004_03-GFP_000.tif.zip](https://github.com/image-js/tiff-js/files/234173/Eimg_000000004_03-GFP_000.tif.zip)

```
identify Cimg_000000008_03-GFP_000.tif
Cimg_000000008_03-GFP_000.tif TIFF 1360x1024 1360x1024+0+0 16-bit Grayscale Gray 2.798MB 0.000u 0:00.000
identify: Unknown field with tag 50838 (0xc696) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/887.
identify: Unknown field with tag 50839 (0xc697) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/887.
```
Would still be nice if we could open them with the library

**Comment by targos** [comment]
what's the error ?

**Comment by targos** [comment]
with tiff-js I mean

**Comment by lpatiny** [comment]
![image](https://cloud.githubusercontent.com/assets/1484241/14779312/e0fa367e-0ad7-11e6-9237-58172bcfc761.png)
But based on the size I think this image is not compressed

**Comment by targos** [comment]
uncompressed is still a compression type (I only support this one xD)

**Comment by targos** [comment]
fixed by 14c13a438b97b12f57e0efbd013630054f7c8125
